src/components/Upload/UploadFile.vue

```diff
@@ -1,23 +1,25 @@
 <template>
-	<div class="space-y-6">
+	<div class="space-y-2 md:space-y-6">
```

Comment:
This doesn't look good - Lot of white space below - what if we streched to the bottom, screen height, and placed the button on the bottom.
![image](https://user-images.githubusercontent.com/27869979/109664644-a2ec9f00-7b6d-11eb-864b-7a889b9b5fa8.png)